### DISCUSSION TITLE
No black/white color option?

### DISCUSSION THREAD

**Discussion by BigCrustyOne** [flameshot-org/flameshot#1946] [Ideas]
Flameshot is almost perfect, but it is missing a critical feature; it needs to offer black and white as color options. Is there some profound reason why this was not included? Is there any reason why it cannot be included in the next release?

**Comment by mmahmoudian** [comment]
Thank you for showing interest in Flameshot. Literally less than 24 hours ago it was _again_ suggested (#1942). Please search in already suggested feature in the issues page before opening a new post.

The "profound reason" is that you can already add it in your config file since about 4 years ago, that simple. Instructions in #1942 as well as a new suggestion.

**Reply by BigCrustyOne** [reply]
Thanks for the feedback. Maybe consider this evidence of how important this feature is to users? Also, thanks for the pointer to custom config edits, although I don't see any 'instructions' for config customization in that thread. That kind of info might be a useful addition to the flameshot.org website? Rather than hiding it here on a programmers forum, adding it to user guide on the main website would provide readily available assistance with requiring forum discussions. Most users are going to go to the app site first for answers...

**Reply by mmahmoudian** [reply]
@BigCrustyOne I forgot where yesterday I posted the answer, but I found multiple instances of it in the issues:

- https://github.com/flameshot-org/flameshot/issues/720#issuecomment-638058112
- https://github.com/flameshot-org/flameshot/issues/1202#issuecomment-754166292

Yes it is not well-documented and we need to improve the documentation. We can use your help if you are interested to improve the documentation on the website and contribute to the project:

1. Fork the repo: https://github.com/flameshot-org/flameshot-org.github.io
2. Apply the changes you think are useful and necessary in your fork (you can also do it on the Github website if that is more comfortable for you)
3. Commit the changes to your fork
4. Go to your fork's "Pull requests" tab
5. Create a pull request

You can find very good visual step-by-step instruction on how to do the steps above in this awesome Github repo:
https://github.com/firstcontributions/first-contributions

**Reply by BigCrustyOne** [reply]
superb. Thanks again and my apologies for the topic duplication. I'll consider helping, but I won't commit unless I can actually contribute. Time is always the issue. Cheers.
